packages/Kokkos/src/DTK_PointCloud_def.hpp

```diff
@@ -105,8 +104,8 @@ class LocalBoundingBoxFunctor
     {
         for ( size_t d = 0; d < 3; ++d )
         {
-            dst[d] = std::numeric_limits<SC>::max();
-            dst[d + 3] = -std::numeric_limits<SC>::max();
+            dst[d] = KokkosHelpers::numericLimitsMax<SC>();
+            dst[d + 3] = -KokkosHelpers::numericLimitsMax<SC>();
```

Comment:
@dalg24 @Rombur @dalg24 should we rely on the Tpetra code in the Kokkos::Details namespace for the arithmetic traits or should we have our own? There's is complete and I would argue it should be moved into Kokkos and out of the Details namespace. I would prefer to use it.